src/components/MenuItem/MenuItem.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from "next/link";
+import styles from "./MenuItem.module.scss";
+
+type Props = {
+  closeMenu: Function;
+  name: String;
+};
+
+export function MenuItem({ closeMenu, name }: Props) {
+  return (
+    <Link
+      href={"/"}
+      style={{
+        textDecoration: "none",
+        fontStyle: "italic",
+      }}
+      as="/"
+    >
+      <li
+        className={styles.menuItem}
+        onClick={() => {
```

Comment:
if you dont need to pass params, you can just input the function directly

```suggestion
        onClick={closeMenu}
```

Reply:
It wont let me do this due to this error message: Type 'Function' is not assignable to type 'MouseEventHandler<HTMLLIElement>'. I am going to leave it for now since we are going to redesign anyways